common/icons/components/Accessibility.tsx

```diff
@@ -0,0 +1,15 @@
+// This is autogenerated from an SVG file
+// Do not edit this file directly; edit the SVG file and run "yarn compile"
+import { SVGProps } from 'react';
+function SvgAccessibility(props: SVGProps<SVGSVGElement>) {
+  return (
+    <svg viewBox="0 0 24 24" {...props}>
+      <path
+        fillRule="evenodd"
```

Comment:
I think there should be `class="icon__shape" fill="currentColor"` in here, to match the others. Maybe there'd be a way to automate it?

Reply:
Thanks, I always forget that bit!

Reply:
Just double checking it's done in the SVG itself as the tsx components are overwritten on each compile.